browser/src/control/Control.SheetsBar.js

```diff
@@ -69,6 +69,9 @@ L.Control.SheetsBar = L.Control.extend({
 		}
 
 		if (id === 'insertsheet') {
+			if (this.map._docLayer.isAutoFillPopupOpen)
```

Comment:
Do we need this if we close the popup on tab change? Also, i can click on the sheet button and add a new sheet. Popup doesn't close in that case. But it closes if we close the popup on tab change.